packages/clarity-js/src/data/metadata.ts

```diff
@@ -144,9 +144,13 @@ function track(u: User, consent: BooleanFlag = null): void {
   // Convert time precision into days to reduce number of bytes we have to write in a cookie
   // E.g. Math.ceil(1628735962643 / (24*60*60*1000)) => 18852 (days) => ejo in base36 (13 bytes => 3 bytes)
   let end = Math.ceil((Date.now() + (Setting.Expire * Time.Day))/Time.Day);
+  // If dob is not set in the user object, use today's date as a dob
+  let date = new Date();
+  let dob = [date.getFullYear(), ('0' + (date.getMonth() + 1)).slice(-2), ('0' + date.getDate()).slice(-2)].join('');
```

Comment:
Wouldn't this be in local time? Want to ensure in UTC. Instead use:

new Date().toISOString().Substring(0, 10).Replace('-','')

or similar. Definitely fix this to use UTC as that is critical.

Reply:
Nice catch, updated the code.